modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="enabling-assisted-installer-service-on-bare-metal_{context}"]
+= Enabling assisted installer service on bare metal
+
+The Assisted Installer Service (AIS) deploys {product-title} clusters. {rh-rhacm-first} ships with AIS, which is deployed when the MultiClusterHub Operator is enabled on the {rh-rhacm} hub cluster.
```

Comment:
When this content GAs, I suggest rewriting the introduction so that the user action of the module is in the first paragraph and the context is in a different module.